spec/classes/server_spec.rb

```diff
@@ -48,6 +49,16 @@
         }
       end
 
+      describe 'when unsetting pid file' do
+        let :params do
+          { :pid_file => 'UNSET' }
+        end
+
+        it {
+          is_expected.not_to contain_concat__fragment('rsyncd_conf_header').with_content(/^pid file\s*=\s*\/var\/run\/rsyncd.pid$/)
```

Comment:
I'd not even check for a value.
```suggestion
          is_expected.not_to contain_concat__fragment('rsyncd_conf_header').with_content(/^pid file\s*=/)
```

The reason is that this would pass on `pid file = UNSET` but you don't `pid file =` to show up at all.

Reply:
Good idea, done!